tests/typescript-mock-data.spec.ts

```diff
@@ -51,36 +60,61 @@ it('should generate mock data functions with external types file import', async
     const result = await plugin(testSchema, [], { typesFile: './types/graphql.ts' });
 
     expect(result).toBeDefined();
-    expect(result).toContain("import { Avatar, UpdateUserInput, User, Status } from './types/graphql';");
+    expect(result).toContain(
+        "import { ABCType, Avatar, UpdateUserInput, User, ABCStatus, Status } from './types/graphql';",
+    );
     expect(result).toMatchSnapshot();
 });
 
 it('should generate mock data with typename if addTypename is true', async () => {
     const result = await plugin(testSchema, [], { typesFile: './types/graphql.ts', addTypename: true });
 
     expect(result).toBeDefined();
-    expect(result).toContain("import { Avatar, UpdateUserInput, User, Status } from './types/graphql';");
```

Comment:
Removed - it's verified by the snapshot and is not directly relevant to the purpose of the test.

Reply:
Well I prefer to have this specific expect, as it's clear by reading the test what it does and what we expect.
Yest the snapshot verifies it, but we have to open the snapshot file, find the right test name and search for this `import` see if the value is correct.  It's more prone to errors, and a little duplication like here helps highlight what we expect

 For me a unit test should be clear enough to not require any other file to be opened.

Reply:
I removed that one because the case in test here is typename, typesFile. I have adjusted the test to use the typename option only and explicitly check for the presence of `__typename`.